main.go

```diff
@@ -142,7 +161,21 @@ func main() {
 		RHOBSSecretName:              fmt.Sprintf("%v-prom-remote-write", addonName),
 		RHOBSEndpoint:                envVars[rhobsEndpointEnvVarName],
 		RHSSOTokenEndpoint:           envVars[rhssoTokenEndpointEnvVarName],
-	}).SetupWithManager(mgr, nil); err != nil {
+		NetworkType:                  controllers.SDNNetworkType,
```

Comment:
NetworkType is a misleading name, please change to something like CNIType or NetworkInterfaceType.

Reply:
Done